devpack/src/main/java/io/neow3j/devpack/Signer.java

```diff
@@ -0,0 +1,71 @@
+package io.neow3j.devpack;
+
+import io.neow3j.devpack.annotations.Instruction;
+import io.neow3j.script.OpCode;
+import io.neow3j.transaction.WitnessScope;
+
+/**
+ * Represents a signer and provides signer-related information. It is returned for example when calling
+ * {@link io.neow3j.devpack.contracts.LedgerContract#getTransactionSigners(Hash256)} .
+ */
+public class Signer {
+
+    /**
+     * The account of the signer.
+     */
+    public Hash160 account;
+
+    /**
+     * The scopes of the witness.
+     */
+    public byte witnessScopes;
+
+    /**
+     * The contracts that are allowed by the witness. Only available when the corresponding signer has a witness scope
+     * {@link WitnessScope#CUSTOM_CONTRACTS}.
+     */
+    public Hash160[] allowedContracts;
+
+    /**
+     * The groups that are allowed by the witness. Only available when the corresponding signer has a witness scope
+     * {@link io.neow3j.transaction.WitnessScope#CUSTOM_GROUPS}.
+     */
+    public ECPoint[] allowedGroups;
+
+    /**
+     * The groups that are allowed by the witness. Only available when the corresponding signer has a witness scope
+     * {@link io.neow3j.transaction.WitnessScope#CUSTOM_GROUPS}.
```

Comment:
Witness scope should be `WITNESS_RULES` not `CUSTOM_GROUPS`

Reply:
I'll change that.